src/components/ux-tooltip/ux-tooltip.scss

```diff
@@ -0,0 +1,62 @@
+/*
+Supposed to use this but _functions.scss is currently not imported
+$small-screen: em-calc(768);
+$medium-screen: em-calc(1280);
+$large-screen: em-calc(1440); */
+
+$small-screen: 768px;
+$medium-screen: 1280px;
+$large-screen: 1440px;
+
+$screen: "only screen";
+$small: "only screen and (min-width: 0px) and (max-width: #{$small-screen})";
+$medium: "only screen and (min-width:#{$small-screen}) and (max-width: #{$medium-screen})";
+$large: "only screen and (min-width:#{$medium-screen})";
+
+@mixin tooltipHover {
+	display: block;
+	width: auto;
+	&:after {
+		content: "x";
+		position: absolute;
+		top: 0;
+	}
+}
+
+*[id*='uxTooltip'] {
```

Comment:
It is a CSS wildcard matching so it will match all tooltips with ID contains "uxTooltip" -- but I will remove this anyway because apparently this is only being used in use cases.